scripts/ci/prek/update_airflow_pyproject_toml.py

```diff
@@ -74,7 +74,7 @@
     "fab": parse_version("2.2.0"),
     "openlineage": parse_version("2.3.0"),
     "git": parse_version("0.0.2"),
-    "common.messaging": parse_version("1.0.3"),
+    "common.messaging": parse_version("2.0.0"),
```

Comment:
Since all providers using common.messaging must have version 2.0.0 due to code refactor

Reply:
Nice!